pkg/scheduler/cache/cache.go

```diff
@@ -232,6 +232,33 @@ type defaultStatusUpdater struct {
 	vcclient   *vcclient.Clientset
 }
 
+func NewPodStatusClient(config *rest.Config) (*kubernetes.Clientset, error) {
+	configShallowCopy := *config
+	qps := os.Getenv("POD_STATUS_CLIENT_QPS")
+	if qps != "" {
```

Comment:
how about qps is not set in env? what's default qps? use kubeclient's default one?

Reply:
if not set env, we will use `config`, which is configed by cmd flag param and consistent with other kube client.